scripts/dtlscid.test

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
On Mac see a warning, but test still works:

```
% ./scripts/dtlscid.test
No matching processes belonging to you were found
No matching processes belonging to you were found
(standard_in) 1: illegal character: N
(standard_in) 1: illegal character: N
Sending CID is 4242
Sending CID is 4141
Cleaning up...
```

Please followup later with adding tests to the CI.